client/app/bundles/course/assessment/submission/pages/SubmissionsIndex/SubmissionsTableRow.jsx

```diff
@@ -306,6 +306,13 @@ const SubmissionsTableRow = (props) => {
               : null}
           </TableCell>
         ) : null}
+        {assessment.hasTimeLimit ? (
```

Comment:
instead of `: null`, why not
```ts
{assessment.hasTimeLimit && (
  <TableCell style={tableCenterCellStyle}>
    {submission.timerStartedAt && formatDate(submission.timerStartedAt)}
  </TableCell>
)}
```

Reply:
done